client/env

```diff
@@ -0,0 +1,2 @@
+JWT_SECRET_KEY="X7#pWq@8!Lm\$Yr92&K^Jo4F%AbZ*tNcQ"
```

Comment:
i guess it's time to delete the repository

Reply:
![man-holding-his-head-with-both-hands-pensative-and-stress-face-G6ATAM](https://github.com/user-attachments/assets/fff655e0-1561-4674-9f10-637bb073645c)